### ISSUE TITLE
What is the purpose of `react-static-routes`

### ISSUE THREAD

**Issue by whmountains** [nozzle/react-static#52]
Is it just a convenience method?  Would there be any adverse effects to using a raw `<Switch>` component instead?

Edit: Which file has the source code for this import?  I would like to read the source for myself.

**Comment by tannerlinsley** [comment]
Nope. That is the automatic routing component described in the docs. If you
don't want to use it, you don't have to. Refer to the custom routing
example to see how.

**Comment by tannerlinsley** [comment]
`react-static-routes` is generated at run time using the routes in your `static.config.js`. Here is where that happens: https://github.com/nozzle/react-static/blob/master/src/static.js#L275

**Comment by tannerlinsley** [comment]
Here is the section from the docs on automatic vs custom routing: https://github.com/nozzle/react-static#automatic-routing-with--routes

**Comment by DanWebb** [comment]
I'm seeing this issue on react static 4.7.1, react router 1.0.0, macOS High Sierra 10.13.2

When I attempt to build it provides the same error `Error: Cannot find module 'react-static-routes'`

My npm scripts look like this
```
  "scripts": {
    "start": "NODE_ENV=development react-static start",
    "build": "react-static build",
    "serve": "serve dist -p 3000"
  },
```
`start` continues to work as expected but `build` is still giving the error after re-installing all the dependancies.

I can see a `dist/react-static-routes.js` file is generated but all that's in `node_modules/react-static-routes` is a package.json file, is that normal?

Let me know if you need any further info on this one.

**Comment by rileylnapier** [comment]
where is the source code for react-static-routes?  is it inside react-static?

**Comment by tannerlinsley** [comment]
It's a dynamically generated file placed in the dist folder during build.
It's resolved via an alias in both the dev and build environments.

**Comment by DanWebb** [comment]
Sorry for not replying sooner I did solve this one, I had to delete react-static-routes out of node_modules and package.json otherwise it pulls in the wrong package, like @tannerlinsley suggests it's a dynamically generated file per build not a module installed through npm.

In my case I incorrectly ran `npm i -S react-static-routes` without thinking much into it because my linter was complaining the package wasn't listed.

I think it may be worth noting in the docs somewhere.
